sol/main.c

```diff
@@ -508,9 +508,9 @@ run_sol(struct net_config *net_conf, struct sol_config *sol_conf)
 		goto out;
 	}
 
-	ret = init_mailbox("sol_reqs", 2 * sol_conf->pri_req_max_len,
-		sizeof(struct priority_req), sol_conf->lcore_id,
-		&sol_conf->mb);
+	ret = init_mailbox("sol_reqs", 31 - __builtin_clz(rte_align32pow2(2 *
+		sol_conf->pri_req_max_len)), sizeof(struct priority_req),
```

Comment:
Shouldn't we drop `31 - __builtin_clz(...)` and leave just `rte_align32pow2(2 * sol_conf->pri_req_max_len)`?

Reply:
You are right, `rte_log2_u32(2 * sol_conf->pri_req_max_len)` is the expression that we want.